policy/release_manager.md

```diff
@@ -0,0 +1,54 @@
+# Release manager role
+
+## Appointment
+
+A release manager (RM) is appointed for a specific upcoming major or minor release (i.e., x.y.z -> x.y+1.0 or x.y.z -> x+1.0.0).
+
+A RM may and should step down at any time, when they cannot fulfill the associated obligations.
+In this case, a new RM is appointed.
+
+The appointment of an RM ends with the declaration of the respective release's end-of-life by the RM.
+
+## Responsibilities
+
+Immediately after the previous release was made, the RM takes over decision making regarding merges into the integration branch for all developments targeting the respective release.
+Such merges can be made as soon as, but no earlier than, the RM indicates approval.
```

Comment:
just to make it clear -- merges could be made by anyone with merge permissions, not necessarily RM, as long as RM indicates approval (but not necessarily clicks "Merge").

It makes procedure vague, i.e. should I monitor my PR and merge it if it is approved, or could I just rely that RM would merge it.  Since it is kinda a procedure now, let's keep it as vague as it is now, can improve later.  On the other hand, why would RM not merge it if already approved?